skins/laika/src/store/payment/actions.ts

```diff
@@ -52,12 +49,12 @@ export const actions = {
 		context.commit( SET_TYPE, payload );
 		context.commit( SET_TYPE_VALIDITY );
 	},
-	[ setAccountId ]( context: ActionContext<Payment, any>, payload: TypeData ): void {
-		context.commit( SET_BANK_ACCOUNT_ID, payload );
-		context.commit( SET_BANK_ACCOUNT_ID_VALIDITY );
+	[ setBankData ]( context: ActionContext<Payment, any>, payload: BankAccountData ): void {
+		//TODO Perform AJAX request
+		context.commit( SET_BANKDATA, payload );
 	},
-	[ setBankId ]( context: ActionContext<Payment, any>, payload: TypeData ): void {
-		context.commit( SET_BANK_ID, payload );
-		context.commit( SET_BANK_ID_VALIDITY );
+	[ setLegacyBankdata ]( context: ActionContext<Payment, any>, payload: BankAccountData ): void {
```

Comment:
When passing the URL in, the code in both actions will be identical. Maybe let the BankData component decide which URL to use and have only one store action?